spec/models/user_spec.rb

```diff
@@ -172,4 +172,27 @@
       expect(another_user.can_receive_deposits_from.to_a).to eq [@subject]
     end
   end
+  describe "class methods" do
+    describe "recent_users" do
+      let(:new_users) { User.all.order(created_at: :desc) }
+
+      before do
+        (1..3).each {|i| User.create(email: "abc#{i}@blah.frg", password: "blarg1234", created_at:DateTime.now - i.days) }
+      end
+
+      it "returns based on start date" do
+        results = User.recent_users(Date.today - 2.days)
+        expect(results.count).to eq 2
+        expect(results).to include(new_users[0],new_users[1])
+        expect(results).not_to include(new_users[2])
+      end
+
+      it "returns based on start date and end date" do
+        results = User.recent_users(Date.today - 2.days, Date.today - 1.days)
```

Comment:
@awead I would need an additional context to set that as the subject.  Is that really more clear?